app/classifier/annotation-renderer/svg.jsx

```diff
@@ -0,0 +1,237 @@
+    const normalizedPoint2 = this.eventCoordsToSVGCoords(e.pageX, e.pageY);
+    difference.x = normalizedPoint2.x - normalizedPoint1.x;
+    difference.y = normalizedPoint2.y - normalizedPoint1.y;
+    return difference;
+  }
+
+  // transforms the event coordinates
+  // to points in the SVG coordinate system
+  eventCoordsToSVGCoords(x, y) {
+    const svg = this.svgSubjectArea;
+    const newPoint = svg.createSVGPoint();
+    newPoint.x = x;
+    newPoint.y = y;
+    const matrixForWindowCoordsToSVGUserSpaceCoords = this.getMatrixForWindowCoordsToSVGUserSpaceCoords();
+    const pointforSVGSystem = newPoint.matrixTransform(matrixForWindowCoordsToSVGUserSpaceCoords);
+    return pointforSVGSystem;
+  }
+
+  render() {
+    let taskDescription;
+    let InsideSubject;
+    const { type, src } = getSubjectLocation(this.props.subject, this.props.frame);
+    const createdViewBox = `${this.props.viewBoxDimensions.x} ${this.props.viewBoxDimensions.y} ${this.props.viewBoxDimensions.width} ${this.props.viewBoxDimensions.height}`;
+
+    if (this.props.annotation.task) {
+      taskDescription = this.props.workflow.tasks[this.props.annotation.task];
+      ({ InsideSubject } = tasks[taskDescription.type]);
+    }
+
+    const svgStyle = {};
+    if (type === 'image' && !this.props.loading) {
+      // Images are rendered again within the SVG itself.
+      // When cropped right next to the edge of the image,
+      // the original tag can show through, so fill the SVG to cover it.
+      svgStyle.background = 'black';
+
+      // Allow touch scrolling on subject for mobile and tablets
+      if (taskDescription) {
+        if (tasks[taskDescription.type] && tasks[taskDescription.type].AnnotationRenderer !== SVGRenderer) {
+          svgStyle.pointerEvents = 'none';
+        }
+      }
+      if (this.props.panEnabled === true) {
+        svgStyle.pointerEvents = 'all';
+      }
+    }
+
+    const svgProps = {};
+
+    const hookProps = {
+      taskTypes: tasks,
+      workflow: this.props.workflow,
+      task: taskDescription,
+      classification: this.props.classification,
+      annotation: this.props.annotation,
+      frame: this.props.frame,
+      scale: this.getScale(),
+      naturalWidth: this.props.naturalWidth,
+      naturalHeight: this.props.naturalHeight,
+      containerRect: this.getSizeRect(),
+      getEventOffset: this.getEventOffset,
+      onChange: this.props.onChange,
+      preferences: this.props.preferences,
+      normalizeDifference: this.normalizeDifference,
+      getScreenCurrentTransformationMatrix: this.getScreenCurrentTransformationMatrix
+    };
+
+    Object.keys(tasks).map((task) => {
+      const Component = tasks[task];
+      if (Component.getSVGProps) {
+        Object.assign(svgProps, Component.getSVGProps(hookProps));
+      }
+    });
+
+    let children = [];
+    const isDrawingTask = taskDescription && (tasks[taskDescription.type].AnnotationRenderer === SVGRenderer);
```

Comment:
For some reason `tasks[taskDescription.type].AnnotationRenderer === SVGRenderer` always evaluates to `false` for all tasks.